h5py/h5pl.pyx

```diff
@@ -0,0 +1,57 @@
+# This file is part of h5py, a Python interface to the HDF5 library.
+#
+# http://www.h5py.org
+#
+# Copyright 2008-2019 Andrew Collette and contributors
+#
+# License:  Standard 3-clause BSD; see "license.txt" for full license terms
+#           and contributor agreement.
+
+"""
+    HDF5 plugin interface.
+"""
+
+include "config.pxi"
+from utils cimport emalloc, efree
+
+# === C API ===================================================================
+
+IF HDF5_VERSION >= (1, 10, 1):
+    cpdef append(const char* search_path):
+        """(STRING search_path)"""
+        H5PLappend(search_path)
```

Comment:
Should these have error handling, or at least a return value so that you can check if they succeeded?

(Again, I'm not that familiar with the low-level code; maybe this is somehow taken care of automatically)

Reply:
I believe it is, due to the way `api_gen.py` works (it adds `except *` to every generated `cdef`).

Reply:
OK, great, I hadn't looked at `api_gen.py`.